napari_animation/utils.py

```diff
@@ -1,8 +1,10 @@
 from .easing import Easing
+from .interpolation import Interpolation, interpolation_dict
 
 
 def interpolate_state(initial_state, final_state, fraction):
```

Comment:
This might make sense to put inside the `interpolation.py` file now, but I don't really mind either way